src/error.rs

```diff
@@ -21,9 +67,46 @@ macro_rules! implement {
             fn erfc(self) -> Self {
                 unsafe { math::erfc(self as f64) as Self }
             }
+
+            #[inline]
+            fn erfinv(self) -> Self {
+                erfinv(self as f64) as Self
+            }
         }
-    )
+    };
 }
 
 implement!(f32);
 implement!(f64);
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use assert;
+
+    // Test values calculated from scipy
+    #[test]
+    fn erfinv_of_zero_should_be_zero() {
+        assert::close(0.0_f64.erfinv(), 0.0, 1E-12);
```

Comment:
Let’s be consistent and use the lowercase _e_ here and below. And why do you need those suffixes?